app/modules/users/permissions/rules.py

```diff
@@ -33,9 +38,42 @@
         'is_researcher',
     ],
     ('AssetGroupSighting', AccessOperation.WRITE_PRIVILEGED, 'Object'): ['is_internal'],
+    ('Encounter', AccessOperation.READ, 'Module'): ['is_privileged', 'is_researcher'],
+    ('Encounter', AccessOperation.WRITE, 'Module'): [
+        'is_active'
+    ],  # TODO is this still correct
+    ('Encounter', AccessOperation.READ, 'Object'): ['is_privileged', 'is_researcher'],
```

Comment:
This can work. We don't need much in the way of security around the encounter since so much is filtered through sighting access.